OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -996,19 +1012,24 @@ public void ResolveOrder(Actor self, Order order)
 				if (order.Target.Type != TargetType.Actor)
 					return;
 
+				var targetActor = order.Target.Actor;
+				var isForceEnter = orderString == "ForceEnter";
+				var canResupplyAt = isForceEnter ? AircraftCanResupplyAt(targetActor, !Info.TakeOffOnResupply) : AircraftCanResupplyAt(targetActor);
```

Comment:
This can simplify to `var canResupplyAt = AircraftCanResupplyAt(targetActor, isForceEnter && !Info.TakeOffOnResupply);`